docs/maintenance/redis.md

```diff
@@ -0,0 +1,234 @@
+
+
+\# Redis Maintenance
+
+
+
+\## Introduction
+
+
+
+When running Airflow with the \*\*CeleryExecutor\*\* or \*\*CeleryKubernetesExecutor\*\*, Redis is often used as the message broker.
+
+Redis stores task-related data, queue information, and other temporary state.
+
+
+
+Over time, Redis can grow very large if keys are not cleaned up.
```

Comment:
@ashb / @kaxil - isn't there a way today to configure TTL for Celery's broker's messages? I wonder how it has never been reported as an issue before #55344.